content/home/examples/a-simple-component.md

```diff
@@ -4,7 +4,7 @@ order: 0
 domid: hello-example
 ---
 
-Τα React components εφαρμόζουν μια μέθοδο `render ()` που λαμβάνει δεδομένα και επιστρέφει τι πρέπει να εμφανίζει. Αυτό το παράδειγμα χρησιμοποιεί μια σύνταξη τύπου XML που ονομάζεται JSX. Τα δεδομένα που μεταφέρονται στο component μπορούν είναι προσβάσιμα στη μέθοδο `render ()` μέσω του `this.props`.
+Τα React components εφαρμόζουν μια μέθοδο `render()` που λαμβάνει δεδομένα και επιστρέφει τι πρέπει να εμφανίζει. Αυτό το παράδειγμα χρησιμοποιεί μια σύνταξη τύπου XML που ονομάζεται JSX. Τα δεδομένα που μεταφέρονται στο component μπορούν είναι προσβάσιμα στη μέθοδο `render()` μέσω του `this.props`.
```

Comment:
@pvinis Thanks a lot! Nice catch 👍
Feel free to contribute in translating a page if you can find some time!
Let me know if I can help in any way!